audit/projects/k8s-infra-prow-build-trusted/services/container/clusters/prow-build-trusted.json

```diff
@@ -18,8 +18,8 @@
   "binaryAuthorization": {},
   "clusterIpv4Cidr": "10.4.0.0/14",
   "createTime": "2020-04-30T23:44:46+00:00",
-  "currentMasterVersion": "1.19.9-gke.1400",
-  "currentNodeVersion": "1.19.9-gke.1400",
+  "currentMasterVersion": "1.19.9-gke.1900",
+  "currentNodeVersion": "1.19.9-gke.1900",
```

Comment:
Noise from cluster auto-upgrades, but I feel like this is useful confirmation that they're actually happening